docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -42,6 +42,12 @@ Dashboard filters are applied to all the widgets in your dashboard. They consist
 
 When adding a widget or editing an existing one, you will be taken to a new page where you can see the dashboard widget builder. The widget builder has several configuration options that you can use to shape data and add information to your dashboard. Learn more in the [Widget Builder documentation](/product/dashboards/widget-builder/).
 
+### Metrics Extraction
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+If you're using custom tags or custom filters for in your widgets, we will start collecting metrics to give you accurate counts and rates. During widget creation or editing you will see approximate data based on indexed events, but as soon as you save we will start extracting metrics for those widgets.
```

Comment:
![Screenshot 2024-02-22 at 3 00 19 PM](https://github.com/getsentry/sentry-docs/assets/6111995/127fd045-54cc-463a-b1da-b2691bf8cdad)

We can maybe add an image?

Reply:
That's a good idea!

Reply:
Yes, agree on adding an image!